packages/core/src/App/Containers/AccountTypesModal/account-types-modal.jsx

```diff
@@ -0,0 +1,553 @@
+                    <FinancialBox mt5OnClick={redirectToMt5Real} />
+                </div>
+                <div label={localize('Demo accounts')}>
+                    <FinancialBox is_demo mt5OnClick={redirectToMt5Demo} />
+                </div>
+            </Tabs>
+        ) : (
+            <div>
+                <FinancialBox mt5OnClick={redirectToMt5Real} />
+            </div>
+        );
+    }
+    throw new Error('Unknown box');
+};
+
+const Box = ({ title, description, footer_text, icons, cards }) => {
+    return (
+        <div className='account-types__box'>
+            <div className='account-types__box-left'>
+                <h2 className='account-types__box-title'>{title}</h2>
+                <p className='account-types__box-description'>{description}</p>
+                {footer_text && <p className='account-types__box-footer'>{footer_text}</p>}
+                <div className='account-types__box-icons'>
+                    {icons.map((icon, index) => {
+                        return <Icon className='account-types__box-icon' icon={icon} key={index} />;
+                    })}
+                </div>
+            </div>
+            {cards}
+        </div>
+    );
+};
+
+Box.propTypes = {
+    title: PropTypes.string,
+    description: PropTypes.string,
+    footer_text: PropTypes.string,
+    icons: PropTypes.arrayOf(PropTypes.string),
+    cards: PropTypes.array,
+};
+
+const FinancialBox = ({ is_demo = false, mt5OnClick }) => {
+    return (
+        <Box
+            title={localize('Financial account ({{type}})', { type: is_demo ? localize('Demo') : localize('Real') })}
+            description={localize(
+                'Financial accounts offer you to trade financial assets such as Forex, commodities and indices.'
+            )}
+            footer_text={localize('Over 50 tradable financial assets')}
+            icons={['IcUnderlyingFRXNZDJPY', 'IcUnderlyingOTC_NDX', 'IcUnderlyingFRXXAUUSD', 'IcUnderlyingFRXBROUSD']}
+            cards={[
+                <AccountCard
+                    key={0}
+                    title={localize('Trade on Deriv')}
+                    subtitle={localize('Option trading account')}
+                    button_text={is_demo ? localize('Add demo account') : localize('Add real account')}
+                    // TODO: Add click handler
+                    buttonOnClick={() => {}}
+                    items={{
+                        [localize('Multiplier')]: localize('Up to X1000'),
+                        [localize('Stop loss')]: localize('Flexible'),
+                        [localize('Take profit')]: localize('Flexible'),
+                        [localize('Cancel Trade')]: localize('Allow'),
+                        [localize('Currency')]: localize('USD/GBP/EUR'),
+                    }}
+                    // TODO: Update paths
+                    learn_more={[
+                        {
+                            text: localize('Option Trading'),
```

Comment:
shouldn't these all be sentence case? I see things like `Cancel Trade` etc as well, as I know our deriv is following sentence case throughout the site

Reply:
@negarn I copied all static texts from Zeplin, but I will fix them by myself